src/ProtocolV3TestBase.sol

```diff
@@ -17,8 +17,8 @@ import { ReserveConfiguration } from 'sparklend-v1-core/contracts/protocol/libra
 import { WadRayMath }           from 'sparklend-v1-core/contracts/protocol/libraries/math/WadRayMath.sol';
 import { DataTypes }            from 'sparklend-v1-core/contracts/protocol/libraries/types/DataTypes.sol';
 
-import { Ethereum } from 'spark-address-registry/src/Ethereum.sol';
-import { Gnosis }   from 'spark-address-registry/src/Gnosis.sol';
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
```

Comment:
Updating the `spark-address-registry` changed the automatic remapping generated by Foundry that was adding the `/src` at the end so we ended up with some modifications because of this.

We do recommend migrating from using git submodules into NPM packages which solve a lot of issues that sometimes happen. Git submodules end up being pretty buggy sometimes. We are ok to tackle this changes on a different PR if agreed!

Reply:
Sounds good. We will stick with submodules for now.